modules/nlu/package.json

```diff
@@ -25,6 +25,8 @@
     "rimraf": "^3.0.2"
   },
   "dependencies": {
+    "fs-extra": "^7.0.1",
```

Comment:
don't need that anymore

Reply:
yes in [model-service.ts](https://github.com/botpress/botpress/blob/dev/modules/nlu/src/backend/model-service.ts)